app/client/stylesheets/_home.scss

```diff
@@ -73,32 +58,40 @@
 		position: absolute;
 	}
 
-	.logo-header {
-		height: 100px;
-		padding: 35px 0 0 50px;
-		position: absolute;
+	#hero-content {
+		height: 100%;
+		position: relative;
+		color: white;
+		z-index: 3;
+	}
+
+	#hero-content>div {
```

Comment:
Since this is a sass file can you put it like 
#hero-content {
   div {
    }

}

Reply:
Fixed